src/lib/rules/no-vulnerable-javascript-libraries/no-vulnerable-javascript-libraries.ts

```diff
@@ -133,6 +134,20 @@ const rule: IRuleBuilder = {
             }
         };
 
+        /** Removes any tags from a version. E.g.: 2.0.0rc --> 2.0.0, 2.0.1-pre --> 2.0.1 */
+        const removeTagsFromVersion = (version: string): string => {
+            const validCharacters = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '.'];
```

Comment:
@molant You can also use `semver for that, e.g.:

```js
const version = 'v3.4.0-rc';

console.log(`${semver.major(version)}.${semver.minor(version)}.${semver.patch(version)}`);
```

Reply:
Awesome 👍